ppr-ui/src/store/state/state-model.ts

```diff
@@ -107,8 +107,8 @@ export const stateModel: StateModelIF = {
         postalCode: ''
       },
       emailAddress: '',
-      phoneNumber: null,
-      phoneExtension: null
+      phoneNumber: '',
+      phoneExtension: ''
```

Comment:
There is some confusion with the type on these properties. 
In another PR there is work to enforce the type on these as `number`.

Unfortunately it DOES matter, ie string is initialized as this, `number` as `null`. 
So i believe this is right, according to the spec.

Reply:
What happens when you test the api call? As Cameron was saying, there is another ticket that ensures it is a number(I am working on that one)...

Reply:
The Api will return a 400 bad request if I pass in the wrong type. In the spec, it's asking for a string.
![image](https://user-images.githubusercontent.com/62360407/186709894-55c36046-07e3-4450-b4c0-f6a28ae9f085.png)